Core/Core/ActAsUser/ActAsUserWindow.swift

```diff
@@ -48,21 +54,6 @@ public class ActAsUserWindow: UIWindow {
         fatalError("init(coder:) has not been implemented")
     }
 
-    public override var rootViewController: UIViewController? {
-        didSet {
```

Comment:
From what I can tell, this isn't really an override at all, and is just adding an extra observer. See https://gist.github.com/cobbal/5b93442e8b5487a119a9c4d9f9711975

Reply:
Does it matter if the original var was implemented in ObjC? Can you still reproduce the issue with this change?

Reply:
Same behavior for ObjC. I can only reproduce it now on iOS 12. Maybe the new modal presentation style doesn't have this issue... which would be great